src/dynamic-page-layout.scss

```diff
@@ -0,0 +1,361 @@
+    padding: $padding-y $padding-x;
+  }
+
+  @mixin box-shadow() {
+    box-shadow: var(--sapContent_HeaderShadow);
+    // box-shadow: 0 0.25rem 0.25rem -0.25rem rgba(0,0,0,0.15), inset 0 -0.0625rem 0 0 var(--sapObjectHeader_BorderColor);
+  }
+
+  @mixin no-shadow {
+    &--no-shadow {
+      box-shadow: none;
+    }
+  }
+  // todo: check for mixin naming conventoin
+  @mixin flex-config($justify-content: space-between) {
+    display: flex;
+    flex-direction: row;
+    justify-content: $justify-content;
+    align-items: center;
+  }
+
+  @mixin custom-button() {
+    @include flex-config($justify-content: space-evenly);
+
+    margin: 0 0.25rem;
+    width: 1.5rem;
+    height: 2rem;
+    min-width: 1.5rem;
+  }
+
+  &__title-area {
+    @include fd-reset();
+
+    @include fd-flex($direction: column) {
+      justify-content: space-between;
+    }
+
+    min-height: 3rem;
+    word-wrap: break-word;
+    background-color: $fd-dynamic-page-layout-background-color; // discrepancy
+
+    @include fd-hover() {
+      background: var(--sapObjectHeader_Hover_Background);    // color not yet available in Fundamental, therefore hover not reflecting the color
+    }
+
+    &--s {
+      @include responsive-page($padding-y: $fd-dynamic-page-layout-padding-top-bottom, $padding-x: 0.5rem);
+    }
+
+    &--m_l {
+      @include responsive-page($padding-y: $fd-dynamic-page-layout-padding-top-bottom, $padding-x: 2rem);
+    }
+
+    &--xl {
+      @include responsive-page($padding-y: $fd-dynamic-page-layout-padding-top-bottom, $padding-x: 3rem);
+    }
+
+    &--transparent-bg {
+      background: transparent;  // WARNING: theme conflicting values
+    }
+
+    &--collapsed {
+      padding-bottom: 1rem;
+    }
+
+    &[aria-hidden="true"] {
+      display: none;
+    }
+  }
+
+  &__title-container {
+    @include fd-reset();
+    @include flex-config($justify-content: flex-start);
+  }
+
+  .#{$fd-namespace}-breadcrumb.#{$block}__breadcrumb {
+    @include fd-reset();
+
+    padding: 0.5rem 0 0.25rem 0;
+
+    &--s {
+      padding: 0;
+    }
+  }
+
+  &__breadcrumb-container {
+    @include fd-reset();
+    @include flex-config();
+  }
+
+  &__title {
+    @include fd-reset();
+
+    color: var(--sapGroup_TitleTextColor);
+    font-size: var(--sapFontHeader3Size);
+    padding: 0.3125rem 0 0 0;
+
+    &--collapsed {
+      @include fd-ellipsis();
+    }
+
+    &--collapsed-s {
+      color: var(--sapGroup_TitleTextColor);
+      font-size: var(--sapFontHeader5Size);
+      padding: 0;
+      line-height: 2rem;
+
+      @include fd-ellipsis();
+    }
+    // tablet spec
+    &--m {
+      padding: 0.6875rem 0 0 0;
+    }
+  }
+
+  &__subtitle {
+    @include fd-reset();
+
+    color: var(--sapContent_LabelColor);
+    font-size: var(--sapFontSize);
+    margin-top: 0.25rem;
+  }
+
+  &__title-content {
+    @include fd-reset();
+
+    padding: 0 0 0 1rem;
+
+    @include fd-rtl() {
+      padding: 0 1rem 0 0;
+    }
+  }
+
+  &__toolbar {
```

Comment:
You are using this class like this:
```
<div class="fd-dynamic-page-layout__toolbar fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
```
If you have a reset it will reset the styling coming from `fd-toolbar`. fd-toolbar comes with a reset already. Your class in this case is a modifier class for `fd-toolbar` and no reset is needed.

Reply:
removed `fd-reset()`